example/tests/util.mojo

```diff
@@ -1,9 +1,20 @@
 import testing
 
 
-fn assert_true(cond: Bool, message: String) raises:
+@value
+struct MojoTest:
     """
-    Wraps testing.assert_true, raises Error on assertion failure.
+    A utility struct for testing.
     """
-    if not testing.assert_true(cond, message):
-        raise Error(message)
+
+    var test_name: String
+
+    fn __init__(inout self, test_name: String):
+        self.test_name = test_name
+        print("# " + test_name)
```

Comment:
This should use TEST_ITEM_PREFIX  from plugin.py

Reply:
We will need to do something like this

```mojo
    fn __init__(inout self, test_name: String):
        self.test_name = test_name
        try:
            let plugin = Python.import_module("pytest_mojo.plugin")
            print(plugin.TEST_ITEM_PREFIX.to_string() + " " + "test_name")
        except:
            print("#" + " " + "test_name")
```

We would be defaulting to # if the Python setup is not right, and we should also print something in the except 😞 .
How often are we planning to change the TEST_ITEM_PREFIX? I see changing it once more because I do not love that it is the comment symbol(#), but that would be it. What do you think?

Reply:
@lleites that's a good point. It seems like overkill especially because the `MojoTest` is just in the example folder, not part of the pytest plugin. I also would be concerned about the repeated importing every time `MojoTest` is imported. I think let's leave it as you have it currently.